tests/qos/files/qos.yml

```diff
@@ -1309,3 +1309,155 @@ qos_params:
                 cell_size: 208
             hdrm_pool_wm_multiplier: 4
             cell_size: 208
+    th3:
```

Comment:
How doe the variables "broadcom_th3_hwskus" gets interpreted as "th3" here?  is there something already handling that?

Reply:
Yes there is an exiting logic which handles it for all ASIC types the same way.